### DISCUSSION TITLE
How to get the already initialized cfg:DictConfig

### DISCUSSION THREAD

**Discussion by wbqtac** [facebookresearch/hydra#2860] [General]
Hi there,

@hydra.main() will create an object cfg:DictConfig, but passing it deeply into the code will cost more effort and need change existing function param list to take in cfg. 

I understand there is function, initialize_config_dir, which is to initialize and load the config again. But as it load the config again, I suppose the dynamic config from command line will be lost.

Is there any way to reach the cfg again in the code, after hydra.main intialized and loaded the configuration?

Best

Bing

**Comment by Jasha10** [comment]
Hi Bing,

A similar question was asked here: https://github.com/facebookresearch/hydra/discussions/2858
